### ISSUE TITLE
Ambiguous wording confuses newbie, film at 11

### ISSUE THREAD

**Issue by seebs** [HellFirePvP/AstralSorcery#1602]
From the spyglass description: "and the user must have at least seen one of these constellations traced out before"

What I thought this meant: You must have seen *at least one* constellation paper, after which you can trace any constellation you see.

What it appears to mean: You must have seen constellation paper *for a specific constellation* in order to draw it and learn it.

**Comment by HellFirePvP** [comment]
I suppose. Though i'd argue that one can only discover something if being given reference material beforehand.

**Comment by seebs** [comment]
Presumably someone somewhere discovered things without reference material, once. It might actually be a sort of interesting minigame to have some kind of feedback for individual lines that hints at whether they're correct, but that would be a very different UI.

It doesn't help that when I couldn't figure out what was happening, and went to look for clues in guides, they were all from before the recent reshuffle of the stars. I actually figured this out by writing down estimated coordinates for the stars I was seeing, and reading the source, and realizing that I had to be looking at Discidia, and that the stars weren't right, then checking commit history and finding out that they'd changed recently. Which meant that I was disregarding older guides, which meant that I didn't notice the thing where they phrased that requirement differently.

I do find myself confused more often by this in-game book than by some, and I'm not sure *why* yet. It's well-written, I just seem to get confused easily.
